distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,27 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+=== Configuration Export/Import
+==== Export Configuration Settings
+
+1) Launch DDF.
+
+2) At the Admin Console, select any Active Application that you want to save.
```

Comment:
@ricklarsen, do we have an existing section somewhere that explains how to configure DDF? If not, do you think we should keep this section very general and say something like _Make all the necessary configuration changes_ or something? I don't think we should explain how to use the Admin UI here.